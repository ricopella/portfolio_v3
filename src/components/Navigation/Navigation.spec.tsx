import React from 'react'
import renderer from 'react-test-renderer'
import { motion } from 'framer-motion'
import { PureNavigation as Navigation } from './index'
import { render } from '@testing-library/react'

describe('<Navigation />', () => {
  it('should match Navigation snapshot', () => {
    const internalTree = renderer
      .create(
        <motion.nav animate={'closed'}>
          <Navigation headerItems={[]} />
        </motion.nav>
      )
      .toJSON()

    expect(internalTree).toMatchSnapshot()
  })
})
