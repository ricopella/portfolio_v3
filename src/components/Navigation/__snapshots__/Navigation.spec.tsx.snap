// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navigation /> should match Navigation snapshot 1`] = `
<nav
  style={Object {}}
>
  <ul
    className="css-1am0w18"
    style={Object {}}
  />
</nav>
`;
